### DISCUSSION TITLE
How to enable seeing prereleases in CKAN

### DISCUSSION THREAD

**Discussion by ProgrammerFailure** [KSP-CKAN/CKAN#4079] [Q&A] [answered]
As per title, I want to be able to see the prerelease versions of mods in CKAN, and have the option to download said prereleases. Is there a mechanism for this?

**Comment by HebaruSan** [comment]
No, there's no setting to show pre-releases in CKAN.

`netkan.exe` has a `--prerelease` flag, though, if you want to roll your own metadata.

**Reply by ProgrammerFailure** [reply]
Where do you get netkan.exe? Also I thought that **only** showed prereleases?

**Reply by HebaruSan** [reply]
> Where do you get netkan.exe?

<https://ksp-ckan.s3-us-west-2.amazonaws.com/netkan.exe>

> Also I thought that **only** showed prereleases?

Yes, that's how the `--prerelease` flag works.

**Reply by ProgrammerFailure** [reply]
What I want is something that shows both prereleases and normal versions. I guess the way to achieve that would be making two repos and combining them somehow?
Edit: ah, I see CKAN allows adding multiple repos. That should work for me.
Edit 2: the netkan.exe file throws an antivirus alert by microsoft defender: https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Trojan%3AWin32%2FWacatac.B!ml&threatid=2147735505
Is there a reason for this?

**Reply by HebaruSan** [reply]
Microsoft Defender is extremely prone to false positives, complaining about a problem where none exists. It happens with virtually every CKAN release before we request whitelisting.

**Comment by HebaruSan** [comment, marked as answer]
Pre-release support was subsequently added in #4260.
